packages/plugin-vue-jsx/index.d.ts

```diff
@@ -1,6 +1,12 @@
 import { Plugin } from 'vite'
 import { VueJSXPluginOptions } from '@vue/babel-plugin-jsx'
+import { FilterPattern } from '@rollup/pluginutils'
 
-declare function createPlugin(options?: VueJSXPluginOptions): Plugin
+declare interface CommonOptions{
+    include?: FilterPattern;
+    exclude?: FilterPattern;
+}
+
+declare function createPlugin(options?: VueJSXPluginOptions & CommonOptions): Plugin
```

Comment:
Don't we want it directly in the `VueJSXPluginOptions`? Why not?

Reply:
I think `VueJSXPluginOptions` is type of `@vue/babel-plugin-jsx`, but `include` / `exclude` is only use to vite.